packages/webdriver/src/utils.js

```diff
@@ -316,6 +316,10 @@ export function getErrorFromResponseBody (body) {
 export class CustomRequestError extends Error {
     constructor(body) {
         super(body.value.message || body.value.class || 'unknown error')
-        if (body.value.error) this.name = body.value.error
+        if (body.value.error) {
+            this.name = body.value.error
+        } else if (body.value.message && body.value.message.includes('stale element reference')) {
```

Comment:
It looks like chrome isn't passing the error property as expected, so we'll sniff it, and hardcode the name if matched

Reply:
Can we add the element/selector properties to the error message. Just how do you expect users to diagnose 👍 
WARN webdriver: Request encountered a stale element - terminating request